TO-DO-APP/app/src/main/java/com/example/to_do_app/MainActivity.kt

```diff
@@ -1,46 +0,0 @@
-package com.example.to_do_app
-
-
-import android.content.Intent
-import androidx.appcompat.app.AppCompatActivity
-import android.os.Bundle
-import androidx.recyclerview.widget.LinearLayoutManager
-import androidx.room.Room
-import kotlinx.android.synthetic.main.activity_main.*
-import kotlinx.coroutines.GlobalScope
-import kotlinx.coroutines.launch
-
-
-// entity - table
-// dao - queries
-// database
-
-class MainActivity : AppCompatActivity() {
-    private lateinit var database: myDatabase
-    override fun onCreate(savedInstanceState: Bundle?) {
-        super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_main)
-        database = Room.databaseBuilder(
```

Comment:
This is very bad practice. Creating Database in both activities. and using global scope everywhere. 
Use viewmodel for querying from DB in viewModel scope.